zerver/views/home.py

```diff
@@ -265,6 +265,13 @@ def desktop_home(request):
     # type: (HttpRequest) -> HttpResponse
     return HttpResponseRedirect(reverse('zerver.views.home.home'))
 
+def apps_view(request):
+    # type: (HttpRequest) -> HttpResponse
+    if settings.ZILENCER_ENABLED:
+        return render(request, 'zerver/apps.html')
+    else:
+        return HttpResponseRedirect('https://zulipchat.com/apps/', status=301)
```

Comment:
(Also I guess this just re-taught me that GitHub notification emails for line comments don't include any of the previous comments they're replying to.)